server/src/main/java/org/elasticsearch/action/support/master/TransportMasterNodeAction.java

```diff
@@ -104,7 +104,7 @@ protected TransportMasterNodeAction(String actionName, boolean canTripCircuitBre
     protected abstract String executor();
 
     /**
-     * @deprecated new implementors should override {@link #read(StreamInput)} and use the
+     * @deprecated new implementers should override {@link #read(StreamInput)} and use the
```

Comment:
Strictly speaking both `implementor` and `implementer` are correct usage, so I'm not sure it's worth the churn to change these.